### PR TITLE
Remove repo_number+color limit

### PR DESCRIPTION
**Pull request by zhimin-z**
Remove the repo limit by autogenerating the color for each curve.

### CODE REVIEW

File: src/components/MainContainer.js [hunk 1]
```diff
@@ -9,7 +9,6 @@ import StatsTable from "./StatsTable";
 import UrlDisplay from "./UrlDisplay";
 import ClosableBadge from "../shared/ClosableBadge";
 import Footer from "./Footer";
-import stargazerLoader, { maxReposAllowed } from "../utils/StargazerLoader";
```
Comment (seladb) [inline src/components/MainContainer.js:12]: You removed `stargazerLoader` which is used in line 81...

File: src/components/RepoPreloader.js [hunk 2]
```diff
@@ -2,7 +2,6 @@ import React, { useState, useEffect, useRef } from "react";
 import PropTypes from "prop-types";
 import { Container, ProgressBar, Button } from "react-bootstrap/";
 import "./RepoPreloader.css";
-import stargazerLoader, { maxReposAllowed } from "../utils/StargazerLoader";
```
Comment (seladb) [inline src/components/RepoPreloader.js:5]: ditto

**Comment by seladb** [comment]
@zhimin-z the build fails because of formatting issues, can you please take a look?

**Comment by zhimin-z** [comment]
> @zhimin-z the build fails because of formatting issues, can you please take a look?

I see, do you have any idea about running prettier? I am unfamiliar with js formatting honestly. @seladb

**Comment by seladb** [comment]
> > @zhimin-z the build fails because of formatting issues, can you please take a look?
> 
> Hi @seladb, is that good now?

It's still failing... you can run `np run format:fix` and it will automatically fix all the formatting errors

**Comment by zhimin-z** [comment]
> npm run format:fix

Done! @seladb

**Comment by seladb** [comment]
@zhimin-z some of the colors are too bright:
image

**Comment by zhimin-z** [comment]
> @zhimin-z some of the colors are too bright: image
> 
> Also - the colors should be more distinguished - the 2 first colors looks almost the same

I reinitiate a new pr to solve this. @seladb https://github.com/seladb/StarTrack-js/pull/53

**Pull request closed without merging**